ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -442,7 +444,13 @@ class RangeSelector {
             options = (
                 chart.options.rangeSelector as RangeSelectorOptions
             ),
-            buttonOptions = options.buttons,
+            langOptions = chart.options.lang,
+            buttonOptions = (
+                options.buttons || this.defaultButtons.map(
```

Comment:
Q: Where is `this.defaultButtons` array defined? I can't find it. Tested also with GUI demo:
![image](https://github.com/user-attachments/assets/0a81d90e-05e7-4977-88a5-930f2c77d290)

Reply:
IIRC there was a refactor here, likely missed when merging in. Will remove the remaining `defaultButtons` references